climesync.py

```diff
@@ -181,6 +182,44 @@
         print
         pp.pprint(ts.get_times(query))
 
+    if choice == "st":
+        project = raw_input("Project slug? ")
+        
+        if not project:
+            print "You must provide a project slug!"
+            continue
+
+        start_date = raw_input("(Optional) Start date (yyyy-mm-dd): ")
+        end_date = raw_input("(Optional) End date (yyyy-mm-dd): ")
+
+        query = dict()
+
+        if project:
+            query["project"] = [p.strip() for p in project.split()]
+        if start_date:
+            query["start"] = [start_date]
+        if end_date:
+            query["end"] = [end_date]
+
+        result = ts.get_times(query)
+
+        if len(result) != 0 and \
+          (not isinstance(result, list) or "pymesync error" in result[0]):
+            pp.pprint(result)
+
+        else:
+            time_sum = 0
+            for user_time in result:
+                time_sum += user_time["duration"]
+
+            hours = time_sum // 3600
+            time_sum -= hours * 3600
+            minutes = time_sum // 60
+            time_sum -= minutes * 60
+            seconds = time_sum
+
+            print "The total time worked on project %s is %d hours %d minutes %d seconds" % (project, hours, minutes, seconds)
```

Comment:
Check out http://share.sm3.su/writing_idiomatic_python_3.pdf section 2.3.1

Actually, this is a great book and everyone who writes python should read it.